app/overrides/jobs/attach_files_to_work_job_override.rb

```diff
@@ -6,10 +6,10 @@
 
   def perform(work, uploaded_files, **work_attributes)
     uploaded_files.each do |uploaded_file|
-      # [hyc-override] check all files for viruses
+      # [hyc-override] check all files for  viruses
```

Comment:
```suggestion
      # [hyc-override] check all files for viruses
```

minor spacing change